ui.frontend/src/view/FormFieldBase.js

```diff
@@ -505,6 +505,75 @@ class FormFieldBase extends FormField {
         }
     }
 
+    /**
+     * Common method to update the enums of checkbox and radiobutton
+     * @param {Array} newEnums - updated enum values
+     * @param {CallableFunction} createItemCallback - function to create options
+     */
+    updateEnumForRadioButtonAndCheckbox(newEnums, createItemCallback) {
```

Comment:
FormFieldBase should only have common functionality across fields, ideally you should create something like FormOptionFieldBase and add all re-usable view code specific to option fields inside it.

Reply:
done!